python/lsst/ip/diffim/getTemplate.py

```diff
@@ -192,10 +199,20 @@ def runQuantum(self, exposure, butlerQC, skyMapRef, coaddExposureRefs):
                                   (dataId['tract'], dataId['patch']))
                     availableCoaddRefs[dataId['patch']] = coaddRef
 
-        patchList = [tractInfo[patch] for patch in availableCoaddRefs.keys()]
-        templateExposure = self.run(tractInfo, patchList, detectorCorners, availableCoaddRefs,
-                                    visitInfo=exposure.getInfo().getVisitInfo())
-        return pipeBase.Struct(exposure=templateExposure, sources=None)
+        if overlappingArea == 0:
+            templateExposure = None
+            pixGood = 0
+            self.log.info("No overlapping template patches found")
+        else:
+            patchList = [tractInfo[patch] for patch in availableCoaddRefs.keys()]
+            templateExposure = self.run(tractInfo, patchList, detectorCorners, availableCoaddRefs,
+                                        visitInfo=exposure.getInfo().getVisitInfo())
+            pixNoData = np.count_nonzero(templateExposure.mask.array
```

Comment:
Separately, it may not need an explicit comment, but it took me a moment to work through the logic of the number of no-data pixels equalling the non-zero elements of the test.